server/events/models/models.go

```diff
@@ -33,12 +34,17 @@ import (
 // Repo is a VCS repository.
 type Repo struct {
 	// FullName is the owner and repo name separated
-	// by a "/", ex. "runatlantis/atlantis", "gitlab/subgroup/atlantis", "Bitbucket Server/atlantis".
+	// by a "/", ex. "runatlantis/atlantis", "gitlab/subgroup/atlantis",
+	// "Bitbucket Server/atlantis", "azuredevops/project/atlantis".
 	FullName string
-	// Owner is just the repo owner, ex. "runatlantis" or "gitlab/subgroup".
-	// This may contain /'s in the case of GitLab subgroups.
-	// This may contain spaces in the case of Bitbucket Server.
+	// Owner is just the repo owner, ex. "runatlantis" or "gitlab/subgroup"
+	// or azuredevops/project. This may contain /'s in the case of GitLab
+	// subgroups or Azure Devops Team Projects. This may contain spaces in
+	// the case of Bitbucket Server.
 	Owner string
+	// Project is juts the project name, only required for Azure Devops.  This will never
+	// have /'s in it.
+	Project string
```

Comment:
Can you explain why you can't utilize FullName, Owner and Name? I would prefer to not have vcs-specific data in this model.

Reply:
Totally understand.  I changed the way I was handling this over time, but happy to remove the struct member.  I'll work on a change for that.